File: src/1_Values_Types_Operators/Operators/operators.cpp
#include <iostream>

int main()
{
  // EQUALITY OPERATORS:
  // ==, !=
  int x = 10;
  char y = 'a';
  std::cout << (x == y) << std::endl;
  std::cout << (x != y) << std::endl;

  // OR, AND ASSIGNMENT OPERATORS:
  // ||, &&
  int a = 10 || 20;
  int b = false || 20;
  int c = 10 && 20;
  int d = false && 20;
  std::cout << a << std::endl;
  std::cout << b << std::endl;
  std::cout << c << std::endl;
  std::cout << d << std::endl;
  return 0;
}

File: src/1_Values_Types_Operators/Operators/operators.js
'use strict';

// eslint-disable-next-line require-jsdoc
function main() {
  // EQUALITY OPERATORS:
  // ===, !==, ==, !=
  const x = 10;
  const y = '10';
  console.log(x === y);
  console.log(x !== y);
  console.log(x == y);
  console.log(x != y);

  // OR, AND ASSIGNMENT OPERATORS:
  // ||, &&
  const a = 10 || 20;
  const b = false || 20;
  const c = 10 && 20;
  const d = null && 20;
  console.log(a);
  console.log(b);
  console.log(c);
  console.log(d);
}

if (require.main === module) {
  main();
}
